coderd/notifications/dispatch/webhook_test.go

```diff
@@ -28,17 +28,15 @@ func TestWebhook(t *testing.T) {
 	t.Parallel()
 
 	const (
-		titleTemplate = "this is the title ({{.Labels.foo}})"
```

Comment:
for reviewer:

`Dispatch` can't render Go templates, it expects already rendered arguments.